pkg/api/daemon/openapi.yaml

```diff
@@ -0,0 +1,122 @@
+# This code is copied from https://github.com/rootless-containers/rootlesskit/blob/master/pkg/api/openapi.yaml
+# The code is licensed under Apache-2.0
+
+openapi: 3.0.3
+info:
+  title: bypass4netnsd API
+  version: 0.0.1
+servers:
+  - url: 'http://bypass4netnsd/v1'
+
+paths:
+  /bypass:
+    get:
+      responses:
+        '200':
+          description: An array of BypassStatus
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BypassStatuses'
+
+    post:
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/BypassSpec'
+      responses:
+        '201':
+          description: BypassStatus
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BypassStatus'
+  
+  /bypass/{id}:
+    delete:
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Null response
+
+components:
+  schemas:
+    Proto:
+      type: string
+      description: "protocol for listening. Corresponds to Go's net.Listen. The strings with \"4\" and \"6\" suffixes were introduced in API 1.1.0."
```

Comment:
Remove "The strings with \"4\" and \"6\" suffixes were introduced in API 1.1.0."